audioplayer.h

```diff
@@ -0,0 +1,34 @@
+#pragma once
+#include <Windows.h>
+#include <MMSystem.h>
+#include <iostream>
+#include <io.h>
+#include <dirent.h>
+#include <vector>
+#include <string>
+#include <algorithm>
+
+#pragma comment(lib, "winMM.lib")
+
+using namespace std;
+
+void CallMciSendString(const string& str);
+
+class AudioPlayer {
+
+ private:
+  string selectedTrackName;
+  string currentVolume = "50";
+  bool isPlaying = false;
+
+ public:
+  vector<string> GetAvailableTracks(const string& folder);
+  void ChangeVolume(const string& volume);
+  void RewindAudio(const string& trackTiming);
+  void CloseAudio();
+  bool ChangeTrack(const int& trackNumber);
```

Comment:
it's useless to make int a reference, copying will result in less overhead. So just `const int trackNumber`. You should only pass by reference variables that can be big, like vector or string.